### PR TITLE
include ``devkitarm-rules`` in ``devkita64_devkitarm``

### PR DESCRIPTION
**Pull request by ThatNerdyPikachu**
fixes building on newer devkitARM versions, as the rules were split into a separate package

### CODE REVIEW

File: devkita64_devkitarm/Dockerfile [hunk 1]
```diff
@@ -2,7 +2,7 @@ FROM devkitpro/devkita64
 
 MAINTAINER Dave Murphy <davem@devkitpro.org>
 
-RUN dkp-pacman -Sy --noconfirm devkitARM && \
+RUN dkp-pacman -Sy --noconfirm devkitARM devkitarm=rules && \
```
Comment (fincs) [inline devkita64_devkitarm/Dockerfile]: Should be `devkitarm-rules`

Comment (ThatNerdyPikachu) [reply devkita64_devkitarm/Dockerfile]: fixed.

**Comment by ThatNerdyPikachu** [comment]
Sidenote, why were these rules split into a different package?

**Comment by fincs** [comment]
The rules and crtls were split into different packages so that we can update them without rolling the entire devkitARM toolchain.

**Comment by ThatNerdyPikachu** [comment]
Ah.

**Pull request merged**